infra/modules/aws-bulk/main.tf

```diff
@@ -32,18 +35,17 @@ resource "aws_s3_bucket" "output" {
 }
 
 module "psoxy-file-handler" {
-  source = "../aws-psoxy-instance"
+  source = "git::https://github.com/worklytics/psoxy//infra/modules/aws-psoxy-instance?ref=v0.3.0-beta.3"
 
   function_name        = "psoxy-${var.instance_id}"
   handler_class        = "co.worklytics.psoxy.S3Handler"
   source_kind          = var.source_kind
   path_to_function_zip = module.psoxy-package.path_to_deployment_jar
   function_zip_hash    = module.psoxy-package.deployment_package_hash
-  path_to_config       = "../../../configs/${var.source_kind}.yaml"
-  api_caller_role_arn  = module.psoxy-aws.api_caller_role_arn
-  api_caller_role_arn_name = module.psoxy-aws.api_caller_role_name
+  path_to_config       = "${var.psoxy_basedir}/configs/${var.source_kind}.yaml"
```

Comment:
should be a full variable passed in, consistent with others.

Reply:
extracted part, but can't really do it as we need the `source_kind` of each.